style.css

```diff
@@ -0,0 +1,2 @@
+h1 { color: rgb(47, 127, 192); }
+ 
```

Comment:
- [ ] @Fsher07  nice job so far The Stylelint linter is not passing. Kindly fix the problem before asking for another review.